File: reverse_word.py
def reverse_word(s: str) -> str:
    words = s.split(" ")
    answer = []

    for word in words:
        chars = list(word)
        left, right = 0, len(chars) - 1

        while left < right:
            temp = chars[left]
            chars[left] = chars[right]
            chars[right] = temp

            left += 1
            right -=1

        answer.append("".join(chars))

    return " ".join(answer)



print(reverse_word("Let's take LeetCode contest"))

File: TwoPointers/reverseWord.js
/**
 * @param {string} s 
 */
function reverseWord(s) {
    const words = s.split(' ');
    const answer = [];

    for (const word of words) {
        let left = 0;
        let right = word.length - 1;

        while (left < right) {
            const temp = word[left];
            word[left] = word[right];
            word[right] = temp; 
        }

        answer.push(word.join(''));
    }

    return answer.join(' ');
}

console.log(reverseWord("Let's take LeetCode contest"))